src/common/components/select-repository-list/index.js

```diff
@@ -125,6 +125,34 @@ export class SelectRepositoryListComponent extends Component {
     return renderedRepos;
   }
 
+  renderRepoAmount() {
+    const { ids, isFetching } = this.props.repositories;
+    const { selectedRepositories } = this.props;
+
+    let reposSelected = false;
+
+    // Return nothing until isFetching completes
+    if (isFetching || ids.length === 0) {
+      return;
+    }
+
+    // Check if any respos have been selected
+    if (selectedRepositories.length > 0) {
+      reposSelected = true;
+    } else {
+      reposSelected = false;
+    }
+
+    // Return selected repos amount and the total
+    return (
+      <strong>
+        { reposSelected &&
```

Comment:
And please put `{ids.length} repos` in a separate line to keep the conditional block above it more separate.